testbook/client.py

```diff
@@ -29,8 +50,6 @@ def _execute_result(cell):
 
     @staticmethod
     def _output_text(cell):
-        if not cell["outputs"]:
-            return
         text = ''
         for output in cell["outputs"]:
```

Comment:
This will blow up if you target a markdown cell because they don't have an output